cmd/network/verification.go

```diff
@@ -415,12 +415,12 @@ func (e *EgressVerification) getSubnetIds(ctx context.Context) ([]string, error)
 
 	// If this is a non-BYOVPC cluster, we can find the private subnets based on the cluster and internal-elb tag
 	if len(e.cluster.AWS().SubnetIDs()) == 0 {
-		e.log.Info(ctx, "searching for subnets by tags: kubernetes.io/cluster/%s=owned and %s=", e.cluster.InfraID(), nonByovpcPrivateSubnetTagKey)
+		e.log.Info(ctx, "searching for subnets by tags: kubernetes.io/cluster/%s and %s=", e.cluster.InfraID(), nonByovpcPrivateSubnetTagKey)
 		resp, err := e.awsClient.DescribeSubnets(ctx, &ec2.DescribeSubnetsInput{
 			Filters: []types.Filter{
 				{
-					Name:   aws.String(fmt.Sprintf("tag:kubernetes.io/cluster/%s", e.cluster.InfraID())),
-					Values: []string{"owned"},
+					Name:   aws.String("tag-key"),
+					Values: []string{fmt.Sprintf("kubernetes.io/cluster/%s", e.cluster.InfraID())},
```

Comment:
I'm _pretty_ sure that all of the values in a `Values` block here get `or`'d together, so I'm keeping these two blocks separate since we want both.

Reply:
I think you're right.  The CLI works that way too, and from the type definition for `Filter`: 

```
	// The filter values. Filter values are case-sensitive. If you specify multiple
	// values for a filter, the values are joined with an OR , and the request returns
	// all results that match any of the specified values.
	
```

So, confirmed, I guess :)